inc/sai.h

```diff
@@ -164,6 +164,26 @@ typedef enum _sai_api_t
      * @brief Extensions range base
      */
     SAI_API_EXTENSIONS_RANGE_BASE = 0x20000000,
+
+    SAI_API_CUSTOM_OTN_RANGE_BASE = 0x30000000,
+    SAI_API_TRANSCEIVER           = 0x30000001, /**< sai_transceiver_api_t */
+    SAI_API_LOGICAL_CHANNEL       = 0x30000002, /**< sai_logical_channel_api_t */
+    SAI_API_OTN                   = 0x30000003, /**< sai_otn_api_t */
+    SAI_API_ETHERNET              = 0x30000004, /**< sai_ethernet_api_t */
+    SAI_API_PHYSICAL_CHANNEL      = 0x30000005, /**< sai_physical_channel_api_t */
+    SAI_API_OCH                   = 0x30000006, /**< sai_och_api_t */
+    SAI_API_LLDP                  = 0x30000007, /**< sai_lldp_api_t */
+    SAI_API_INTERFACE             = 0x30000008, /**< sai_interface_api_t */
+    SAI_API_OPTICAL_PORT          = 0x30000009, /**< sai_optical_port_api_t */
+    SAI_API_OA                    = 0x3000000a, /**< sai_oa_api_t */
+    SAI_API_OSC                   = 0x3000000b, /**< sai_osc_api_t */
+    SAI_API_APS                   = 0x3000000c, /**< sai_aps_api_t */
+    SAI_API_APS_PORT              = 0x3000000d, /**< sai_aps_port_api_t */
+    SAI_API_ATTENUATOR            = 0x3000000e, /**< sai_attenuator_api_t */
+    SAI_API_WSS                   = 0x3000000f, /**< sai_wss_api_t */
+    SAI_API_MEDIA_CHANNEL         = 0x30000010, /**< sai_media_channel_api_t */
+    SAI_API_OCM                   = 0x30000011, /**< sai_ocm_api_t */
+    SAI_API_OTDR                  = 0x30000012, /**< sai_otdr_api_t */
```

Comment:
why you are adding those at this specific range ? and not as regualr object? did we agreed on this approach ?

Reply:
that would be good, i think we can discuss what should final decision here be